### ISSUE TITLE
猪耳朵识别软件3

### ISSUE THREAD

**Issue by de7ga** [de7ga/ze#3]
from tensorflow keras models import Sequential
from tensorflow keras layers import Conv2D, MaxPooling2D, Flatten, Dense
 
def create_pig_ear_classifier(input_shape=(224, 224, 3)):
    model = Sequential([
        Conv2D(32, (3, 3), activation='relu', input_shape=input_shape),
        MaxPooling2D((2, 2)),
        Conv2D(64, (3, 3), activation='relu'),
        MaxPooling2D((2, 2)),
        Conv2D(128, (3, 3), activation='relu'),
        MaxPooling2D((2, 2)),
        Flatten(),
        Dense(128, activation='relu'),
        Dense(1, activation='sigmoid')  # 二分类：是/不是猪耳朵
    ])
    
    model compile(optimizer='adam',
                  loss='binary_crossentropy',
                  metrics=['accuracy'])
    return model
 
# 示例模型创建（需要实际数据训练）
# model = create_pig_ear_classifier()
# model fit(X_train, y_train, epochs=10, validation_data=(X_val, y_val))
